### ISSUE TITLE
Issue with loading the pix2pix generator model during test

### ISSUE THREAD

**Issue by ajaykmr2292** [junyanz/pytorch-CycleGAN-and-pix2pix#427]
I am able to train the pix2pix generator model successfully. But when I run the test.py file, its throwing error 

`ConvTranspose2D has no attribute 'model'`. Can you please help me with the reason? Also, I am training two different variants of the model on two different gpus. So, is there any problem because of that?

Thanks in advance.

**Comment by junyanz** [comment]
Does it work for the training and test script on provided datasets (e.g., facades, maps)? It's hard to tell without knowing more details about your training and test.

**Comment by ajaykmr2292** [comment]
Sorry for not conveying the info about my dataset earlier. I used custom training and test set. I used combine_A_and_B.py to create the dataset in the desired format. When I just train one model at a time on a GPU, the test.py seems to work well. But when I train two models at the same time (on two different GPUs in the same PC by giving different GPU ids..like giving `--gpu_ids 0` for the first model and `--gpu_ids 1` for the second model), the training seems to complete. But when I run test.py, I get the above error. 
So, I would like to know whether the model is getting messed up because of some global variables sharing by the two models (or any similar kind of issue)? And is it right to train two models on a single GPU at the same time (Specifically I would like know whether GPU visualizes memory for every single process)?  I am using NVIDIA GeForce GTX 1080 btw.

**Comment by junyanz** [comment]
I don't see a problem here. Maybe you can try to train a model with `--gpu_ids 0` first, and then test it. Then you train the second model with `--gpu_ids` and then test it. See if it works. I don't think two separate jobs will affect each other.
